src/components/Admin/index.jsx

```diff
@@ -211,7 +235,7 @@ class Admin extends React.Component {
     return (
       <Link to={path} className="reset btn btn-sm btn-outline-primary ml-3">
         <Icon className="fa fa-undo mr-2" />
-        Reset to {this.getMetadataForAction().title}
+        Reset to {this.getMetadataForAction(enterpriseId).title}
```

Comment:
No longer need to pass `enterpriseId` as a function argument (see other comment).